### ISSUE TITLE
Failed to merge

### ISSUE THREAD

**Issue by LianAFei** [hhrutter/pdfcpu#74]
hi @hhrutter
When I use the function MergeCommand in this great great project,something wrong with the result.
Use the test files, I could get the merged file,but There is no merged file after used the function with my pdf files. 
And It shows me "DereferenceName: wrong type <false>" with the command pdfcpu.exe merge.
There are the files which failed to merge.
[test4.pdf](https://github.com/hhrutter/pdfcpu/files/3037464/test4.pdf)
[test5.pdf](https://github.com/hhrutter/pdfcpu/files/3037465/test5.pdf)
[test6.pdf](https://github.com/hhrutter/pdfcpu/files/3037467/test6.pdf)

Hope your replay.

**Comment by hhrutter** [comment]
Thank you for reporting this. I will check it out.

**Comment by hhrutter** [comment]
The *InfoDict* has an attribute called *Trapped*. This should be a *Name object*.
In the files you provided *Trapped* is a *Boolean*. Here is the part of the spec that explicitly forbids *Boolean*:

<img width="956" alt="Screenshot 2019-04-04 at 00 13 04" src="https://user-images.githubusercontent.com/11322155/55517750-bf29ed80-5671-11e9-9f4e-f9c1cbe1cc9c.png">

Since all your files open in Acrobat Reader is extended *relaxed validation*.

Long story short - You are fine if you build from the latest commit.
;)

```
Go-> ~/Documents/pdf/out: pdfcpu merge out.pdf test?.pdf
merging into out.pdf: [test4.pdf test5.pdf test6.pdf]
merging in test5.pdf ...
merging in test6.pdf ...
writing out.pdf ...
```

Have fun with pdfcpu!
💚

**Comment by LianAFei** [comment]
Thanks for your beautiful work.@hhrutter
;)
